### ISSUE TITLE
VoiceOver navigation loop on Order Details

### ISSUE THREAD

**Issue by joshheald** [woocommerce/woocommerce-ios#5022]
## Description
When navigating the Order Details screen using VoiceOver and swiping between elements, the navigation loops back to the start after the Product Total field, which leaves much of the screen difficult to access (not impossible: directly tapping on later elements allows access.)

Additionally, the ℹ button and ⌄ related to showing information about Shipping Labels do not have accessibility labels, so are only announced as "button" by VoiceOver.

## To Reproduce

https://user-images.githubusercontent.com/2472348/134023123-148e0090-9e7a-418b-b0c8-366dcbb10584.mp4

1. Go to Orders
2. Tap on an order without fees
3. Enable VoiceOver
4. Swipe through controls until you get to Product Total
5. Continue swiping
6. Observe that you're returned to the top of the screen

## Expected behavior
It should be possible to swipe beyond Product Total through the entire screen

## Mobile Environment
Please include:
- Device: iPhone (7 plus, simulated 12 mini)
- iOS version: 14.6
- WooCommerce iOS version: 7.5.0.0

**Comment by shiki** [comment]
Related to: https://github.com/woocommerce/woocommerce-ios/issues/4977
